### PR TITLE
add tridesclous docker image

### PR DESCRIPTION
**Pull request by magland**
Note that in the container, tridesclous is installed from my fork of that repo. That's because of the issue raised in this PR:
https://github.com/tridesclous/tridesclous/pull/163

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Hi Jeremy. I will try a release very very soon for tridesclous.
Maybe we can wait before merging this.

**Comment by magland** [comment]
> Hi Jeremy. I will try a release very very soon for tridesclous.
> Maybe we can wait before merging this.

That's fine with me. But FYI, the image is already on dockerhub so that I can use it with spikeforest. Hopefully that won't confuse anyone.

File: tridesclous/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.8
+
+RUN pip install numpy
+
+# Install prerequisites
+RUN apt-get update && apt-get install -y libgl1-mesa-glx
+RUN pip install Cython
+RUN pip install scipy numpy pandas scikit-learn matplotlib seaborn tqdm openpyxl PyQt5==5.14.0 pyqtgraph==0.10 quantities neo numba hdbscan
```
Comment (samuelgarcia) [inline tridesclous/Dockerfile:8]: You have to remove PyQt5==5.14.0 pyqtgraph==0.10

**Comment by samuelgarcia** [comment]
Hi @magland.
Can you test with tridesclous==1.6.1.
Note that you have to remove PyQt5 and pyqtgraph from the docker.
Now it can eb run without GUI.
Tell me if it works.

**Comment by magland** [comment]
@samuelgarcia  Is 1.6.1 on pypi?

**Comment by samuelgarcia** [comment]
Yep.
Since 10min.
https://pypi.org/project/tridesclous/

**Comment by magland** [comment]
@samuelgarcia. It seems to work.
I removed the qt stuff and I had to `pip install loky`.
I did get a warning "Problem import matplotlib.pyplot as plt" but it seemed to run okay.
I updated the docker image on spikeforest repo, and so the example there is now using tdc 1.6.1.
See: https://github.com/flatironinstitute/spikeforest/blob/main/examples/sorting/test_tridesclous.sh

**Comment by magland** [comment]
@samuelgarcia , if you think we are good, you can merge this PR

**Comment by samuelgarcia** [comment]
Yes "Problem import matplotlib.pyplot as plt" is dirty fix because matplotlib try to use Qt5.
So I made a bad "try except " which is enought for the moment.

**Pull request merged**